package.json

```diff
@@ -32,7 +32,7 @@
     "syncpack": "^10.1.0"
   },
   "engines": {
-    "node": "^16.11.0 || ^18.0.0 || ^20.0.0",
+    "node": "^16.11.0 || ^18.0.0 || ^20.0.0 || ^22.0.0",
```

Comment:
As an aside, I wonder if we should simply adopt nodes support model or a modified version of it, e.g. we only support active node LTS. Then at most we'll probably have 3 versions here.

Reply:
Although some Node version may be in maintenance but they are still within Life-of-support timeline. IMO, when we do upgrade, we can simply drop the node versions that has no longer support officially. That would also leave us with around 2-3 version here